docs/src/pages/[platform]/components/avatar/examples/DefaultAvatarExample.tsx

```diff
@@ -0,0 +1,19 @@
+import { Avatar, Flex } from '@aws-amplify/ui-react';
+import { FiSmile } from 'react-icons/fi';
+
+export default function DefaultAvatarExample() {
+  return (
+    <Flex direction="row">
+      {/* Avatar with image */}
+      <Avatar src="/cats/5.jpg" />
+      {/* Avatar with default placeholder icon */}
+      <Avatar />
+      {/* Avatar with initials */}
+      <Avatar>DB</Avatar>
+      {/* Avatar with custom icon */}
+      <Avatar>
+        <FiSmile style={{ width: '60%', height: '60%' }} />
+      </Avatar>
```

Comment:
Just clarifying that for our use case with AIConversation we will just pass two separate instances of this `Avatar` with a custom icon for the user and the assistance right?

Reply:
Yup